### PR TITLE
Add new features for parsing edit summaries for wikidata

### PR DESCRIPTION
**Pull request by Ladsgroup**
Bug: https://phabricator.wikimedia.org/T170835

### CODE REVIEW

**Comment by halfak** [comment]
I can't find a good reason why not to merge this -- but I am worried about the apparent (but clearly not real) drop in fitness.  I'd like to take a little bit of time to figure out what happened.

**Comment by halfak** [comment]
Oh!  Something weird is going on

```
Table:
	         ~False    ~True
	-----  --------  -------
	False      1227      395
	True        192     2451
```
The numbers in that table should add up to 20k.

**Comment by Ladsgroup** [comment]
```
ladsgroup@ores-misc-01:~/editquality/datasets$ wc -l wikidatawiki.*
   24342 wikidatawiki.autolabeled_revisions.20k_2015.json
   24481 wikidatawiki.balanced_revisions.20k_2015.json
    4283 wikidatawiki.human_labeled_revisions.5k_2016.json
    4283 wikidatawiki.labeled_revisions.20k_2015.json
    4265 wikidatawiki.labeled_revisions.w_cache.20k_2015.json
```

I deleted everything and started over again, still  "wikidatawiki.labeled_revisions.20k_2015.json" is 5k instead of 20k. It seems the make file rule or merge_labels utility is broken.

**Comment by Ladsgroup** [comment]
Fixed it, there was a bug somewhere.

File: editquality/feature_lists/wikidatawiki.py [hunk 1]
```diff
@@ -115,8 +162,14 @@ class items:
...
+    comment_uppercase_ratio,
+    comment_numbers_ratio,
+    comment_whitespace_ratio
+]
 damaging = mediawiki.comment + mediawiki.user_rights + \
            mediawiki.protected_user + wikibase.parent + wikibase.diff + \
-           local_wiki
+           local_wiki + comment_related
 reverted = damaging
 goodfaith = damaging
```
Comment (halfak) [inline editquality/feature_lists/wikidatawiki.py:175]: This changes the features for reverted/damaging/goodfaith models but it only rebuilds the damaging model.

**Comment by halfak** [comment]
Blindly trusting binaries.  (Well, I did ask about them in IRC, but they are binaries, so here we are)

**Pull request merged**